tests/fixtures/rules/prefer-destructured-store-props/invalid/test01-errors.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "message": "Destructure bar from store $foo for better change tracking & fewer redraws",
+    "line": 2,
+    "column": 2,
+    "suggestions": [
+      {
+        "desc": "Using destructuring like $: ({ bar } = $foo); will run faster",
+        "messageId": "fixUseDestructuring",
+        "output": "<!-- prettier-ignore -->\n{$: ({ bar } = $foo);\nbar}\n{$foo[bar]}\n\n<!-- eslint-disable-next-line dot-notation -- tests -->\n{$foo.bar}\n"
```

Comment:
This suggestion looks strange to me.

```svelte
{$: ({ bar } = $foo);
bar}
{$foo[bar]}

<!-- eslint-disable-next-line dot-notation -- tests -->
{$foo.bar}
```

I think that at least reactive statements should be placed inside `<script>`.

Reply:
I wasn't sure if it was worth writing all the logic to find/create a `<script>` for that. If you think it is, I can probably figure it out.